scripts/benchmarks.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+steps=20
+repeat=10
+statemineOutput=./runtime/statemine/src/weights/
+statemintOutput=./runtime/statemint/src/weights/
+statemineChain=statemine-dev
+statemintChain=statemint-dev
+
+./target/release/statemint benchmark \
```

Comment:
You can surely use bash loops here. an array of pallets and just having the commend once is much better IMO